### PR TITLE
fix(components/media_modal): Aspect ratio

### PR DESCRIPTION
**Pull request by sorin-davidoi**
A bit over my head with the CSS here. This should fix the issues mentioned by @unarist in #2889.

### CODE REVIEW

File: app/javascript/styles/components.scss [hunk 1]
```diff
@@ -1281,6 +1281,10 @@
   }
 }
 
+.media-modal .react-swipeable-view-container {
+  align-items: center;
```
Comment (unarist) [inline app/javascript/styles/components.scss]: Is not this line superseded by `animateHeight` option?

Comment (sorin-davidoi) [inline app/javascript/styles/components.scss]: Right, but it's superseded by `opacity: 0` (did not know how else to position the canvas behind the image).

**Comment by unarist** [comment]
I'm worried about vertical moving on switching between different aspect ratios, but I don't have ideas yet...

<details>
<summary>GIF</summary>

test
</details>

**Review by Gargron — APPROVED** [review]

**Comment by Gargron** [comment]
@unarist That gif looks pretty cool to me? Except that it seems a part of the old image is visible on the very left?

**Comment by sorin-davidoi** [comment]
@unarist  We could just use the old package (`react-swipeable`), it's small enough.

**Comment by unarist** [comment]
Maybe no one cares about it except me :)

**Pull request merged**